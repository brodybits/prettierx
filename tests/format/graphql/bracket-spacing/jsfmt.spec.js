// [prettierx merge update from prettier@2.3.1 ...]
// using bogus option to keep snapshot more consistent with Prettier 2.3.1
run_spec(__dirname, ["graphql"], { bogus2: null });
run_spec(__dirname, ["graphql"], { bogus1: null, graphqlCurlySpacing: false });
