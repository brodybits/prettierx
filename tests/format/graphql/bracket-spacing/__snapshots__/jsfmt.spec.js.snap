// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bracket_spacing.graphql - {"bogus1":null,"graphqlCurlySpacing":false} format 1`] = `
====================================options=====================================
bogus1: null
graphqlCurlySpacing: false
parsers: ["graphql"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(
    count: {type: "Int", defaultValue: 20}
  ) {
  name
}

=====================================output=====================================
fragment Visit on HighlightedVisit
@argumentDefinitions(count: {type: "Int", defaultValue: 20}) {
  name
}

================================================================================
`;

exports[`bracket_spacing.graphql - {"bogus2":null} format 1`] = `
====================================options=====================================
bogus2: null
parsers: ["graphql"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(
    count: {type: "Int", defaultValue: 20}
  ) {
  name
}

=====================================output=====================================
fragment Visit on HighlightedVisit
@argumentDefinitions(count: { type: "Int", defaultValue: 20 }) {
  name
}

================================================================================
`;
